.github/workflows/fuzzing.yml

```diff
@@ -0,0 +1,75 @@
+name: Fuzzing
+
+on:
+  schedule:
+    # Daily midnight fuzzing
+    - cron: '0 0 * * *'
+
+jobs:
+  fuzzing:
+    name: Fuzzing
+
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+        go-version: [ 1.14.x ]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: ${{ matrix.go-version }}
+
+    # - name: Checkout code
+    #   uses: actions/checkout@v2
+
+    # TODO env GOBIN may not be set
+    # TODO should have if condition to disable for forks?
+    - name: Download go-fuzz tools and the Fuzzit CLI, move Fuzzit CLI to GOBIN
+      # if: isFork == false
+      run: |
+        # Install Clang-7.0 because other versions seem to be missing the file libclang_rt.fuzzer-x86_64.a
+        sudo add-apt-repository "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-7 main"
+        wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+        sudo apt update && sudo apt install -y clang-7 lldb-7 lld-7
+
+        go get -v github.com/dvyukov/go-fuzz/go-fuzz github.com/dvyukov/go-fuzz/go-fuzz-build
+        wget -q -O fuzzit https://github.com/fuzzitdev/fuzzit/releases/download/v2.4.77/fuzzit_Linux_x86_64
+        chmod a+x fuzzit
+        mv fuzzit $(GOBIN)
+
+    # TODO $(Build.Reason) is probably not set
```

Comment:
This depends on how we set up having local-regression vs. scheduled fuzzers. It's just needed to figure out what should the var `fuzz_type` be.

Reply:
I think we use `${{ github.event }}` for this, after a bit of reading.